### DISCUSSION TITLE
jedisClient.expire() deprecated

### DISCUSSION THREAD

**Discussion by anil181** [xetorthio/jedis#2597] [Q&A] [answered]
jedisClient.expire(key, expiry); 

We see this warning for the method expire(byte[], int) from the type BinaryJedisCommands is deprecated.

Any suggestion on alternative?

Thanks,
Anil

**Comment by sazzad16** [comment, marked as answer]
Try expire(byte[], long) method. E.g. 
jedisClient.expire(key, (long) expiry);

**Reply by anil181** [reply]
Thank you, you are always so quick :)

**Reply by sazzad16** [reply]
@anil181 Pleasure.

BTW, how are you using Jedis? With Maven? I'm asking because this info should be available in JavaDoc. If it wasn't in the JavaDoc, please let us know your Jedis version.

**Reply by anil181** [reply]
With Gradle, Using Jedis 3.6.0. Was looking up in the JavaDoc but was unable to locate.

**Reply by sazzad16** [reply]
It's available in JavaDoc. 🤔
